Telegram/SourceFiles/platform/win/specific_win.cpp

```diff
@@ -546,5 +547,27 @@ void psSendToMenu(bool send, bool silent) {
 }
 
 bool psLaunchMaps(const Data::LocationPoint &point) {
+	IApplicationAssociationRegistration *aar = nullptr;
+
+	const auto hr = CoCreateInstance(
+		CLSID_ApplicationAssociationRegistration,
+		nullptr,
+		CLSCTX_INPROC,
+		__uuidof(IApplicationAssociationRegistration),
+		reinterpret_cast<void**>(&aar));
+
+	if (SUCCEEDED(hr)) {
```

Comment:
I'm not really sure, but perhaps it makes sense to add an additional check for `nullptr`?
```suggestion
	if (SUCCEEDED(hr) && aar) {
```

Reply:
I think aar can never be `nullptr` if hr succeeded, but better safe than sorry

Reply:
btw, there doesn't seem to be such checks in the rest of the codebase, e.g.
https://github.com/telegramdesktop/tdesktop/blob/dev/Telegram/SourceFiles/platform/win/windows_app_user_model_id.cpp#L170-L172